app/assets/javascripts/hqmf_util.js.coffee

```diff
@@ -712,7 +712,7 @@ class CrossProduct extends Array
 
 # Create a CrossProduct of the supplied event lists.
 XPRODUCT = (eventLists...) ->
-  hqmf.SpecificsManager.intersectAll(new CrossProduct(eventLists), eventLists)
+  hqmf.SpecificsManager.intersectAll(new CrossProduct(eventLists), eventLists, false, null, considerLeftMost: true)
```

Comment:
What are the situations where we wouldn't want to "considerLeftMost"? Should this be added to all `intersectAll` calls potentially?